.github/workflows/release.yml

```diff
@@ -36,13 +36,13 @@ jobs:
     - name: Publish to test.pypi.org
       if: >-  # "create" workflows run separately from "push" & "pull_request"
         github.event_name == 'release'
-      uses: pypa/gh-action-pypi-publish@master
+      uses: pypa/gh-action-pypi-publish@v1
```

Comment:
So such version.

Reply:
@webknjaz On https://github.com/ansible-community/ansible-lint-action/tags we have the moving tag but I do not remember how we implemented it. But it seems to be working as expected so far.

Reply:
That's not desired. I just wanted to point out that you need to fix it here.